tests/Makefile

```diff
@@ -606,4 +609,10 @@ test-decompress-partial : decompress-partial decompress-partial-usingDict
 	@echo "\n ---- test decompress-partial-usingDict ----"
 	./decompress-partial-usingDict$(EXT)
 
+test-freestanding: freestanding
+	@echo "\n ---- test freestanding ----"
+	./freestanding$(EXT)
+	-strace ./freestanding$(EXT)
```

Comment:
Quick question about this test :
What is the purpose of calling `strace` and `ltrace` ?
Is it informational only ?
The presence of `-` in front of these lines makes it seems so.

Was there an initial intention to make these tests able to fail, if some condition of the executable was not met ?
For example, if `ltrace` returns anything?

Reply:
It's just for information.

Since `strace` and `ltrace` are not POSIX compatible command (right?), I put `-` for them.
I belive Linux and some BSD-like OSes have them, but not sure about wider compatibility.